### ISSUE TITLE
container:restart fails to find classpath resource

### ISSUE THREAD

**Issue by OleksandrBezhan** [earldouglas/xsbt-web-plugin#200]
When I run container:restart then it fails to find classpath resource in the following code:
```
object MySingleton {
  lazy val myProps: Properties  = {
    debug(s"this: $this")
    val r = getClass.getResourceAsStream("/my.properties")
    debug(s"resource: $r")
    r
  }
 }
```
When I run app first time with `container:start` then it works ok and prints following:
```
this: com.example.MySingleton$@618c7c8c
resource: sun.net.www.protocol.jar.JarURLConnection$JarURLInputStream@287479f0
```
But when I run `container:restart` I get `null` for resource:
```
this: com.example.MySingleton$@78b0a870
resource: null
```

**Comment by earldouglas** [comment]
> Upgrade to 1.0.0 solves the issue

Closing, since this issue is resolved.  I've filed the issue of speeding up `container:start` in #236.

**Comment by earldouglas** [comment]
@OleksandrBezhan: Check out [quickstart mode][1], now available in xsbt-web-plugin 2.3.0.

[1]: https://github.com/earldouglas/xsbt-web-plugin/blob/2.3.0/docs/2.3.md#quickstart-mode

**Comment by OleksandrBezhan** [comment]
@earldouglas thanks man I will check it out. Looks awesome.
